### PR TITLE
docs: add cli validate

### PR DESCRIPTION
**Pull request by rephus**
Closes: #5327

### Description:

https://deploy-preview-5654--peaceful-bassi-cbf284.netlify.app/guides/cli/how-to-use-lightdash-validate

### CODE REVIEW

**Comment by rephus** [comment]
I'm not sure how to add this doc to the sidebar...

**Review by IrakliJani — APPROVED** [review]

File: docs/docs/guides/cli/how-to-use-lightdash-validate.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import AutomaticallyDeployChangesGithub from './../../snippets/github-secrets.mdx';
+
+# Lightdash Validate
+
+You can trigger a validation on a project from the CLI, this will allow you to block a release (using Github actions)
```
Comment (TuringLovesDeathMetal) [inline docs/docs/guides/cli/how-to-use-lightdash-validate.mdx:5]:
```suggestion
You can check to see if your changes break any of the content in your project using `lightdash validate`. This command will run the validator which will return a list of content in your project that is broken and details about the errors. 

You can trigger a validation on a project using the Lightdash CLI so you can check locally if your changes will break anything. You can also add `lightdash validate` to your [GitHub Actions](/guides/cli/how-to-use-lightdash-validate.mdx/configure-github-actions) so changes can't be merged unless they pass the validation. 
```

**Review by TuringLovesDeathMetal — CHANGES_REQUESTED** [review]
This is really great @rephus!

Just one small recommendation for some copy change 👀 

Also, for the "GitHub actions" bit - do you have to do anything beyond adding that bit of text? Or would you need to do something else to block the PR (like add a command or something that says `if failed, then`)

File: docs/docs/guides/cli/how-to-use-lightdash-validate.mdx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+```
+# Trigger validation for any project UUID 
+lightdash validate --project <project uuid> 
+``` 
+
+![CLI validate](./assets/cli-validate.png)
+
+
+## Configure Github actions
```
Comment (TuringLovesDeathMetal) [inline docs/docs/guides/cli/how-to-use-lightdash-validate.mdx:28]: Can we add a bit here that references the docs on how to set up GitHub actions? I think there are docs where we talk about the templates? 🤔 

Or maybe we actually need to create a separate reference doc just for GitHub actions + all the actions you can do (I can create a separate issue for that). 
 

In the meantime though, maybe we can add something like this at the bottom of the section?:

```
To learn more about setting up GitHub actions for Lightdash, check out the docs here..
```

**Review by TuringLovesDeathMetal — APPROVED** [review]
Nice! looks great 😚 👌

**Pull request merged**